server/controllers/PropertyController.js

```diff
@@ -15,18 +16,35 @@ cloudinary.config({
 });
 
 const { response } = Respond;
+
+// db helpers  funcs
+const { queryAll, find } = DbHelper;
+
 class PropertyController {
   // view all properties
-  static viewAllProperties(req, res) {
-    return response(res, 200, properties);
+  static async viewAllProperties(req, res) {
+    const { error, response: result } = await queryAll('properties');
+    if (error) {
+      return response(res, 404, error, true);
+    }
+    const { rows, rowCount } = result;
+    if (rowCount > 0) {
+      return response(res, 200, rows);
+    }
+    return response(res, 200, 'no properties found!');
   }
 
   // view specific property
-  static viewPropertyById(req, res) {
+  static async viewPropertyById(req, res) {
     const { id } = req.params;
-    const property = properties.find(item => item.id === parseInt(id, 10));
-    if (property) {
-      return response(res, 200, property);
+    const { error, response: result } = await find('properties', 'id', parseInt(id, 10));
+    if (error) {
+      return response(res, 200, error, true);
```

Comment:
If there's an error, I don't think you want to return status 200. Find an appropriate status code for it.